src/Components/WorkoutplansPage/WorkouPlanMuscles.js

```diff
@@ -0,0 +1,11 @@
+import MusclesMap from './MusclesMap';
+
+export default function WorkoutPlanMuscles({ day }) {
+  return (
+    <>
+      {day.muscles.map(function (muscle, index) {
+        return <MusclesMap muscle={muscle} index={index} />;
```

Comment:
The `key` always needs to be where the map is:
```suggestion
        return <MusclesMap muscle={muscle} key={index} />;
```
Not withing the component.

Reply:
i will change it, Thx